src/resources/postcss/base/forms/_sliders.pcss

```diff
@@ -5,90 +5,103 @@
 	 *
 	 * Example (Horizontal):
 	 * <div class="tribe-common-form-control-slider">
-	 *   <input id="sliderOne" type="range" min="0" max="100" value="50" />
-	 *   <label for="sliderOne">Slider One</label>
+	 *   <input
+	 *     class="tribe-common-form-control-slider__input"
+	 *     id="sliderOne"
+	 *     type="range"
+	 *     min="0"
+	 *     max="100"
+	 *     value="50"
+	 *   />
+	 *   <label class="tribe-common-form-control-slider__label" for="sliderOne">Slider One</label>
 	 * </div>
 	 *
 	 * Example (Vertical):
 	 * <div class="tribe-common-form-control-slider tribe-common-form-control-slider--vertical">
-	 *   <label for="sliderOne">Slider One</label>
-	 *   <input id="sliderOne" type="range" min="0" max="100" value="50" />
+	 *   <label class="tribe-common-form-control-slider__label" for="sliderOne">Slider One</label>
+	 *   <input
+	 *     class="tribe-common-form-control-slider__input"
+	 *     id="sliderOne"
+	 *     type="range"
+	 *     min="0"
+	 *     max="100"
+	 *     value="50"
+	 *   />
 	 * </div>
 	 *
 	 * ----------------------------------------------------------------------------- */
 
-	.tribe-common-form-control-slider {
+	/* .tribe-common-form-control-slider {} */
 
-		input[type="range"] {
-			appearance: none;
-			border: none;
-			cursor: pointer;
-			display: inline-block;
-			margin: 0;
-			padding: 0;
-			width: 120px;
-			vertical-align: middle;
+	.tribe-common-form-control-slider__input {
+		appearance: none;
+		border: none;
+		cursor: pointer;
+		display: inline-block;
+		margin: 0;
+		padding: 0;
+		width: 120px;
+		vertical-align: middle;
 
-			/* -----------------------------------------------------------------------------
-			 * Track styles
-			 * ----------------------------------------------------------------------------- */
-			&::-webkit-slider-runnable-track {
-				@mixin form-control-slider-track;
-			}
+		/* -----------------------------------------------------------------------------
```

Comment:
All of these nested comment blocks look off on their tabbing for the 2nd & 3rd lines?

Reply:
yup, fixed!